docs/articles_en/openvino-workflow/model-optimization-guide/weight-compression.rst

```diff
@@ -71,8 +72,8 @@ in OpenVINO IR using NNCF:
 Now, the model is ready for compilation and inference.
 It can be also saved into a compressed format, resulting in a smaller binary file.
 
-**4-bit weight quantization** method stands for an INT4-INT8 mixed-precision weight quantization,
-where INT4 is considered as the primary precision and INT8 is the backup one.
+**4-bit weight quantization** method stands for an INT4-INT8_ASYM mixed-precision weight quantization,
```

Comment:
Looks strange a bit as you use an abstract notation ("INT4") for 4-bit quantization and NNCF-spicific notation ("INT8_ASYM") for 8-bit.

Reply:
One way to fix this is to say "INT4-INT8" first time and disclose how it maps to NNCF modes.